dev/breeze/doc/adr/0004-using-docker-images-as-test-environment.md

```diff
@@ -120,7 +120,7 @@ suited for the job:
 * management and automated synchronisation of the environment can be easily done using simple, external
   scripts that will check the "freshness" of the images and propose corrective actions (pulling or
   rebuilding the image) for the user whenever they are needed (all that in cross-platform way)
-* we can use specific base image (Debian Bullseye with Long Term Support) and rely on security fixes provided
+* we can use specific base image (Debian bookworm with Long Term Support) and rely on security fixes provided
```

Comment:
Should we use capitalize name for Debian release? E.g. Debian Bookworm.
Also it related to the other places

Reply:
Yeah. It all should be Capitalized name - this is how they are used in https://wiki.debian.org/LTS for example. But the small problem about it that it is lowercase when reported by lsb_release and /etc/os-* .. So not in all cases we can change it to Capitalized version and it has to be manually replaced

Reply:
Fixed